app/javascript/src/utils/index.js

```diff
@@ -1,10 +1,19 @@
-import camelCase from "lodash/camelCase";
 import * as R from "ramda";
 
+const toCamelCase = str =>
+  str
+    .split(" ")
+    .map((word, i) => {
+      if (i === 0) return word.toLowerCase();
+
+      return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+    })
+    .join("");
```

Comment:
So what will be the output if the label is `Ux-Designer` or `UX_designer` or `UX    designer` etc? My expectation is that in all those cases the output should be `uxDesigner`.

Reply:
I just thought of spaces for now, it will not work for the other two cases.

Reply:
I have handled the other two cases here [#20](https://github.com/nitinprojects/wheel/pull/20/files)